src/Microsoft.VisualStudio.Threading.Analyzers/CommonInterest.cs

```diff
@@ -43,5 +55,21 @@ public SyncBlockingMethod(IReadOnlyList<string> containingTypeNamespace, string
 
             public string AsyncAlternativeMethodName { get; private set; }
         }
+
+        internal struct LegacyThreadSwitchingMethod
+        {
+            public LegacyThreadSwitchingMethod(IReadOnlyList<string> containingTypeNamespace, string containingTypeName, string methodName)
+            {
+                this.ContainingTypeNamespace = containingTypeNamespace;
+                this.ContainingTypeName = containingTypeName;
+                this.MethodName = methodName;
+            }
+
+            public IReadOnlyList<string> ContainingTypeNamespace { get; private set; }
```

Comment:
Why? Public fields violate fxcop rules so I tend to avoid them unless there's a compelling reason for it.